src/twisted/conch/checkers.py

```diff
@@ -482,7 +482,7 @@ def __init__(self, keydb):
         self._keydb = keydb
 
     def requestAvatarId(self, credentials):
-        d = defer.maybeDeferred(self._sanityCheckKey, credentials)
+        d = defer.execute(self._sanityCheckKey, credentials)
```

Comment:
I guess that this is private API and people should not mess with it and try to return a deferred.